src/workflow/QueueManager.py

```
@@ -0,0 +1,284 @@
+        Returns:
+            True if successfully canceled
+        """
+        if not self.is_available:
+            return False
+
+        try:
+            from rq.job import Job
+
+            job = Job.fetch(job_id, connection=self._redis)
+            job.cancel()
+            return True
```

Ruff (0.15.4)

[warning] Consider moving this statement to an `else` block

(TRY300)